### ISSUE TITLE
Merge with libphonenumber-csharp?

### ISSUE THREAD

**Issue by ArashMotamedi** [jarroda/libphonenumber-dotnet#3]
Hello and thanks for this port.

Any chance you'd be able to merge your code with the other repo, [libphonenumber-csharp](https://github.com/aidanbebbington/libphonenumber-csharp/)? It seems Aidan who's been maintaining that code hasn't had a chance to do a .Net Core port ([Issue 6](https://github.com/aidanbebbington/libphonenumber-csharp/issues/6)), and if this library merges with that one and publish through Nuget, it'd be great! 

Thanks,
Arash

**Comment by brunoblank** [comment]
Awesome work!

The source needs to be updated (the xml files, they are from june-2016). 

I agree with Arash, the best option is to merge it back to the other repo.

Thanks
Bruno

**Comment by jarroda** [comment]
Yeah I originally forked this from libphonenumber-csharp with the intention of merging it back in.  I was worried the original project wasn't being maintained, but it seems like it is so thats good.  I'll try to make some time within the next week or so to update this and get a PR submitted.

**Comment by RichiCoder1** [comment]
We just ran into this. Let me know if you'd like any help.

**Comment by jarroda** [comment]
I just realized that I actually based my fork on a different repo (there are apparently several of these out there).  Mine was based on https://github.com/erezak/libphonenumber-csharp, not https://github.com/aidanbebbington/libphonenumber-csharp/.

The latter definitely seems to be the only one that is getting metadata updates, so I'll just manually port it instead.  I think I'll just drop my current codebase and switch this repo to aidanbebbington's fork.  

I ported the main `PhoneNumbers` project last night, so the next phase is to see about the test project.

**Comment by jarroda** [comment]
Done. https://github.com/aidanbebbington/libphonenumber-csharp/pull/22

Though I had to create a new fork based on the other repo, so any further work I do will be over there. https://github.com/jarroda/libphonenumber-csharp
